manifests/params.pp

```diff
@@ -81,4 +81,15 @@
   $repo_yumcode = "el${::operatingsystemmajrelease}"
   $repo_gpgcheck = false
   $repo_gpgkey = undef
+
+  # tomcat.conf paramaters
+  $tomcat_cfg_loaded = 1
+  $tomcat_base = '/var/lib/tomcats/'
```

Comment:
I don't have a running server to check, but should this be ```/var/lib/tomcat```? singular or plural? with or without a trailing slash?

Reply:
Its correct:
```
[root@centos7-devel ~]$ grep TOMCATS_BASE /etc/tomcat/tomcat.conf 
# be constructed by joining TOMCATS_BASE and NAME.
TOMCATS_BASE="/var/lib/tomcats/"

Reply:
That comment implies things break if there's no trailing slash.